docs/sources/next/using-k6/http-requests.md

```diff
@@ -66,6 +66,13 @@ The [http module](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-htt
 | [put()](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-http/put)         | Issue an HTTP PUT request.                                                |
 | [request()](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-http/request) | Issue any type of HTTP request.                                           |
 
+## Follow redirects
+
+By default, k6 automatically follows a set number of redirects before giving up on a request and erroring out. You can customize this by using:
```

Comment:
```suggestion
By default, k6 automatically follows a set number of redirects before stopping and returning the last response. You can customize this by using:
```

It doesn't error out it just returns the last response. In the same way yon set redirects to 0, and you will get the response without any redirections. Which will allow you to parse the url to be redirected to from it and follow it ... if you want for example. Potentially letting you do stuff in between redirects.